libr/magic/mdump.c

```diff
@@ -44,8 +44,8 @@
 #define SZOF(a)	(sizeof(a) / sizeof(a[0]))
 
 #ifndef COMPILE_ONLY
-void file_mdump(struct r_magic *m) {
-	static R_TH_LOCAL const char optyp[] = { FILE_OPS };
+void file_mdump(struct r_magic *m, const char *optyp[]) {
```

Comment:
rollback this function signature change as its not necessary to pass this variable because its not modified anywhere, and you can just use the FILE_OPS define directly